cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -43,21 +43,80 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
     // `appToggles` tells us whether a feature is toggled on or off in the current environment.
     if (appToggles.mediaPlayer.enabled) {
       describe('Media Player', () => {
-        it('should render a placeholder image', () => {
-          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-            ({ body }) => {
-              // `video` blocks can also contain audio.
-              const media = getBlockData('video', body);
-              if (media) {
-                cy.get('div[class^="StyledVideoContainer"]').within(() => {
-                  cy.get('amp-img')
-                    .should('have.attr', 'src')
-                    .should('not.be.empty');
-                });
-              }
-            },
-          );
-        });
+        // This test is being temporarily throttled to the service 'news'.
+        if (service === 'news') {
+          it('should have a Media Player', () => {
```

Comment:
Is this a new test? I can't seem to find the Canonical equivalent.

Reply:
Its canonical equivalent was this test `should have a visible play button and duration` but then again in AMP we cannot test for this due to its autoplay nature. Hence only check for the presence of a media Player. This was agreed upon by a sync we had with Rhys and James B

Reply:
Do you mean just check there is there `StyledVideoContainer` and omit L56-65?

Reply:
@jamesbrumpton thoughts?

Reply:
Unless I'm reading it incorrectly, I'd agree with @simonsinclair (based on the architecture of the video player/iframe on AMP). I'd make two suggestions to the above though but please feel free to disregard.

- Would it make sense be a bit more specific with regards to which video it is we're checking? For example, adding `.eq(0)` to the end of the line suggested by Simon.
- If this test is just looking for the readiness of the video player, is there much point if we have a test that's going to check for that the component is ready/has loaded *and* plays?

Finally, looking at the AMP media page on Test, the autoplay issue looks to no longer be an issue.

Let me know your thoughts on the above and whether we feel a call would be of use to discuss rather than constant back and forth via comments. This ticket has been in flight for a while so it'd be good to agree a solution and get it wrapped up sooner rather than later.